.github/workflows/CI.yml

```
@@ -34,15 +34,33 @@ jobs:
           REACT_APP_WASM_COMPILER_URL: 'https://ci.test'
           REACT_APP_RRT: ''
 
-  build-api:
+  test-gear-api:
     runs-on: ubuntu-latest
+
     steps:
       - uses: actions/checkout@v2
-      - name: Use node 16.x
+
+      - name: Install NodeJS 16.x
         uses: actions/setup-node@v2.1.5
         with:
           node-version: 16.x
-      - run: |
-          cd api/api
+
+      - name: Prepare
+        run: |
+          wget -O ./gear-node https://github.com/gear-tech/gear/releases/download/build/gear-node
+          mkdir api/test/wasm
+          wget -O api/test/wasm/examples.tar.gz https://github.com/gear-tech/gear/releases/download/build/examples.tar.gz
+          chmod +x gear-node
+          ./gear-node --dev --tmp &
```

We usually add some delay after running the node. But no need if there are no problems.

https://github.com/gear-tech/gear/blob/master/.github/workflows/CI.yaml#L142

```suggestion
          ./gear-node --dev --tmp & sleep 10
```